src/vdaf.rs

```diff
@@ -123,7 +168,13 @@ pub trait Aggregator: Vdaf {
     type PrepareStep: Clone + Debug;
 
     /// The type of messages exchanged among the Aggregators during the Prepare process.
-    type PrepareMessage: Clone + Debug;
+    type PrepareMessage: Clone + Debug + Decode<Self::PrepareMessageDecodingParameter> + Encode;
+
+    /// Contextual parameters needed for decoding messages into `Self::PrepareMessage`.
+    type PrepareMessageDecodingParameter;
```

Comment:
Similar comment as above: If I have a `PrepareStep`, then I necessarily know how to decode a `PrepareMessage`.